circle.yml

```diff
@@ -0,0 +1,10 @@
+machine:
+  environment:
+    AWS_REGION: 'ap-northeast-1'
+
+test:
+  post:
+    - mvn package
+    - mkdir $CIRCLE_ARTIFACTS/plugin
+    - mv /home/ubuntu/bitbucket-branch-source-plugin/target/*.hpi $CIRCLE_ARTIFACTS/plugin
+    - mv /home/ubuntu/bitbucket-branch-source-plugin/target/*.jar $CIRCLE_ARTIFACTS/plugin
```

Comment:
This was needed for us to build in our environment since we need to use plugin with this fix.  Just the BitbucketServerAPIClient.java change is needed.  But this got pulled into PR.